Sources/Sentry/SentryFileManager.m

```diff
@@ -501,6 +508,16 @@ - (NSArray *)readPreviousBreadcrumbs
     return breadcrumbs;
 }
 
+#pragma mark - Contexts
+
+- (void)moveContextFileToPreviousContextFile
+{
+    SENTRY_LOG_DEBUG(@"Moving context file to previous context file");
+    @synchronized(self.contextFilePath) {
+        [self moveState:self.contextFilePath toPreviousState:self.previousContextFilePath];
+    }
+}
```

Comment:
`m`: Can we add a unit test to the `SentryFileManagerTests` for this method, please?